recipe/meta.yaml

```diff
@@ -44,18 +43,19 @@ test:
     - pytest >=6.1
```

Comment:
We now have pytest-httpserver on main. You can add the dependency https://github.com/frostming/unearth/blob/0.17.5/pyproject.toml#L57 and removed the skipped test.